vendor/bat-native-ads/src/bat/ads/internal/notifications.cc

```diff
@@ -97,15 +106,17 @@ void Notifications::CloseAll() const {
   DCHECK(is_initialized_);
 
   for (const auto& notification : notifications_) {
-    auto id = notification.first;
-    ads_client_->CloseNotification(id);
+    ads_client_->CloseNotification(notification.id);
   }
+  //the diff with RemoveAll: it doesn't save state
```

Comment:
Please change remove so the function takes a should_save flag instead of two functions. We can then remove comment and closeall function

Reply:
Removed.